src/ezdxf/addons/drawing/frontend.py

```diff
@@ -711,12 +711,26 @@ def draw_image_entity(self, entity: DXFGraphic, properties: Properties) -> None:
                 )
 
             elif show_filename_if_missing:
```

Comment:
I'm not sure whether the filename takes on the transformation of the image (eg rotating, mirroring, stretching etc) as my example file does not have examples of that. This implementation will apply the transformation and will ignore clipping as well